src/main/scala/org/constellation/primitives/Genesis.scala

```diff
@@ -154,3 +167,13 @@ object Genesis {
           .map(tcd => dao.transactionService.put(tcd, ConsensusStatus.Accepted))
     }.toList.sequence.void.unsafeRunSync()
 }
+
+object GenesisObservationWriter {
+
+  final val FILE_NAME: String = "genesisObservation"
+
+  def writeToDisk(genesisObservation: GenesisObservation, path: File)(implicit ec: ExecutionContext): IO[Try[File]] =
```

Comment:
you can convert Try to IO, also there should be an explicit shift to bounded thread pool in my opinion